src/middlewared/middlewared/plugins/system.py

```diff
@@ -152,6 +155,328 @@ def debug(self, job):
         return dump
 
 
+class GeneralSystemService(ConfigService):
```

Comment:
General System looks a bit backward.

How about SystemGeneralService then we can use `system.general` namespace.